client/my-sites/site-settings/jetpack-sync-panel/index.jsx

```diff
@@ -171,12 +171,12 @@ class JetpackSyncPanel extends React.Component {
 				<div className="jetpack-sync-panel__action">
 					{ translate(
 						'Jetpack Sync keeps your WordPress.com dashboard up to date. ' +
-							'Data is sent from your site to the WordPress.com dashboard regularly to provide a faster experience. '
+							'Data is sent from your site to the WordPress.com dashboard regularly to provide a faster experience.'
```

Comment:
Hm, why are we removing this space? Seems like this is the source of the problem in the first screenshot?